packages/trader/src/index.ts

```diff
@@ -1,33 +1,136 @@
 import { config } from "dotenv";
-config();
+
 import retry from "async-retry";
+const { getWeb3 } = require("@uma/common");
+const { getAbi } = require("@uma/core");
+const {
+  GasEstimator,
+  Networker,
+  Logger,
+  createReferencePriceFeedForFinancialContract,
+  createTokenPriceFeedForFinancialContract,
+  waitForLogger,
+  delay,
+  DSProxyManager
+} = require("@uma/financial-templates-lib");
+
+const { RangeTrader } = require("./RangeTrader");
+const { createExchangeAdapter } = require("./exchange-adapters/CreateExchangeAdapter");
 import { TraderConfig } from "./TraderConfig";
+config();
+
+export async function run(logger: any, web3: any): Promise<void> {
```

Comment:
logger might be another thing that we could type explicitly -- winston almost certainly has a logger type that we can pull in here.

Reply:
indeed it does! I've imported from `Winston.Logger`